ivre/db/__init__.py

```diff
@@ -242,8 +242,10 @@ def __init__(self):
         try:
             import argparse
             self.argparser = argparse.ArgumentParser(add_help=False)
+            USING_ARGPARSE = True
         except ImportError:
-            self.argparser = utils.FakeArgparserParent()
```

Comment:
1. Is there a reason why you no longer use the wrapper `utils.FakeArgparserParent()`
2. You do not define `USING_ARGPARSE = False`, which will cause the script to crash at line 265.